packages/calico/extra/dcos-calico-libnetwork-plugin.service

```diff
@@ -8,7 +8,7 @@ EnvironmentFile=/opt/mesosphere/etc/calico/calico-node.env
 EnvironmentFile=/opt/mesosphere/etc/calico/calico-node-datastore.env
 ExecStart=/opt/mesosphere/bin/start-calico-libnetwork-plugin.sh
 ExecStartPost=/opt/mesosphere/bin/create-calico-docker-network.py
-TimeoutStartSec=60s
+TimeoutStartSec=180s
```

Comment:
Can we replace it with pre check? I understand now we wait for `cos-calico-libnetwork-plugin.service`, right?
 https://github.com/dcos/dcos/blob/6d4aad5c3b99ba5d7ce4f4d0e0921cf96fb2e901/packages/calico/extra/dcos-calico-libnetwork-plugin.service#L11

Reply:
`create-calico-docker-network.py` can take 120+ seconds to complete, which is why the timeout needs to be increased.

I think it was created this slightly weird way because the libnetwork plugin is the service that needs to be restarted if it exits. The Docker network cannot be created until the plugin is running, which is why it is done as a 'post' step.

It looks like there must be a better way, but I'm going for the simple fix right now.